src/components/FormAlertWrapper.js

```diff
@@ -51,38 +51,50 @@ const defaultProps = {
 //
 // This component takes other components as a child prop. It will then render any wrapped components as a function using "render props",
 // and passes it a (bool) isOffline parameter. Child components can then use the isOffline variable to determine offline behavior.
-const FormAlertWrapper = props => (
-    <View style={props.containerStyles}>
-        {props.isAlertVisible && (
-            <View style={[styles.flexRow, styles.alignItemsCenter, styles.mb3]}>
-                <Icon src={Expensicons.Exclamation} fill={colors.red} />
-                <View style={[styles.flexRow, styles.ml2, styles.flexWrap, styles.flex1]}>
-                    {!_.isEmpty(props.message) && props.isMessageHtml && <RenderHTML html={`<muted-text>${props.message}</muted-text>`} />}
+class FormAlertWrapper extends React.Component {
+    renderMessage() {
```

Comment:
This is missing a JSDoc with the `@returns`, but would recommend reverting most of these changes. They are unrelated to the current changes (I'm having trouble telling exactly what changed about this file otherwise) and I think we normally recommend to keep as much JSX inline as possible (though I'm not sure if this is established somewhere in the style guide or just clash of personal preference).